.github/workflows/containerize.yml

```
@@ -0,0 +1,43 @@
+name: Containerize
```

```suggestion
name: kind image
```
or something else that is more descriptive